src/main/java/de/tum/in/www1/artemis/service/connectors/localci/dto/LocalCIBuildJobQueueItem.java

```diff
@@ -8,26 +8,50 @@ public class LocalCIBuildJobQueueItem implements Serializable {
     @Serial
     private static final long serialVersionUID = 1L;
 
-    private Long participationId;
+    private String name;
+
+    private long participationId;
 
     private String commitHash;
 
-    private Long expirationTime;
+    private long expirationTime;
+
+    private long submissionDate;
 
     private int retryCount;
 
-    public LocalCIBuildJobQueueItem(Long participationId, String commitHash, int retryCount) {
+    private long buildStartDate;
+
+    // 1-5, 1 is highest priority
+    private int priority;
+
+    private long courseId;
+
+    public LocalCIBuildJobQueueItem(String name, long participationId, String commitHash, long submissionDate, int priority, int retryCount, long buildStartDate, long courseId) {
+        this.name = name;
         this.participationId = participationId;
         this.commitHash = commitHash;
         this.expirationTime = 0L;
```

Comment:
not your change but you could delete this imho, `long`s are always initialized with 0 in Java imho